app/pages/sections/order-items/additional-services/dashboard/controller.js

```diff
@@ -26,21 +24,11 @@ export const getAdditionalServicesPageContext = async ({
 };
 
 export const putAdditionalServices = async ({ orderId, accessToken }) => {
-  const putAdditionalServicesEndpoint = getEndpoint({ api: 'ordapi', endpointLocator: 'putAdditionalServices', options: { orderId } });
-  try {
-    const body = {
-      status: 'complete',
-    };
+  const result = await putOrderSection({
+    orderId,
+    sectionId: 'additional-services',
+    accessToken,
+  });
 
-    await putData({
-      endpoint: putAdditionalServicesEndpoint,
-      body,
-      accessToken,
-      logger,
-    });
-    return { success: true };
-  } catch (err) {
-    logger.error(`Error updating additional-services for ${orderId}`);
-    throw new Error();
-  }
+  return result;

```

Comment:
How do you feel about removing this function and just using the putOrderSection directly?

Reply:
I did consider this but thought it was neater as the get and the put are contained within the controller & the route doesn't need to know the section id key. It will result in less code & was torn at the time. Will get this done 👍